<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrivacyTabBase</class>
 <widget class="QWidget" name="PrivacyTabBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>479</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="labelRememberDocuments">
     <property name="text">
      <string>Remember opened documents:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioRememberAllApplications">
         <property name="text">
          <string>For a&amp;ll applications</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioDontRememberApplications">
         <property name="text">
          <string>&amp;Do not remember</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioRememberSpecificApplications">
         <property name="text">
          <string>O&amp;nly for specific applications</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelKeepHistory">
       <property name="text">
        <string>Keep history</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinKeepHistory"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClearRecentHistory">
       <property name="text">
        <string>Clear recent history</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="viewBlacklistedApplicationsContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBlacklistAllNotOnList">
     <property name="text">
      <string>Blacklist all applications not on this list</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
